create_k3s_sysext.sh

```diff
@@ -29,6 +29,11 @@ rm -rf "${SYSEXTNAME}"
 mkdir -p "${SYSEXTNAME}"/usr/local/bin
 curl -o "${SYSEXTNAME}/usr/local/bin/k3s" -fsSL "${URL}"
 chmod +x "${SYSEXTNAME}"/usr/local/bin/k3s
+pushd "${SYSEXTNAME}"/usr/local/bin/
+ln -s ./k3s kubectl
+ln -s ./k3s ctr
+ln -s ./k3s crictl
+popd
```

Comment:
Is this also needed for the rke2 extension?

Reply:
I am currently checking this...

Reply:
Apparently not:
https://get.rke2.io/ contains no hint. And the rke2 shell script unpacks a tarball, which apparently does not contain the links.